### PR TITLE
feat: add uml diagrams

### PR DESCRIPTION
**Pull request by vimalk78**
Generated class diagrams from source for better understanding of kepler-model-server

### CODE REVIEW

**Comment by vimalk78** [comment]
Cc: @sunya-ch @rootfs

File: umls/README.md
Comment (sthaha) [inline umls/README.md:1]: Could we move this under `docs/developer/`

**Comment by vimalk78** [comment]
> This would be very helpful for developer. Thank you. Could you please move the directory as @sthaha suggested? After that, I think this PR is all good to merge.

@sunya-ch  moved

**Review by sunya-ch — APPROVED** [review]
Thank you!

**Pull request merged**